homeassistant/components/mqtt/climate.py

```diff
@@ -470,9 +470,7 @@ def handle_climate_attribute_received(
         except ValueError:
             _LOGGER.error("Could not parse %s from %s", template_name, payload)
 
-    def prepare_subscribe_topics(
-        self, topics: dict[str, dict[str, Any]]
-    ) -> None:  # noqa: C901
+    def prepare_subscribe_topics(self, topics: dict[str, dict[str, Any]]) -> None:  # noqa: C901
```

Comment:
This with the comment causes a longer line. We should have moved the noqa comment instead of having this formatted.